everyvoice/tests/regression/format-coverage.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+"""
+For some reason, the coverage reports in regression tend to include the full paths.
+This script filters them to just start with everyvoice/... and realigns the header
+and footer accordingly.
+
+Usage: coverage report --include='*/everyvoice/*' | python format-coverage.py
```

Comment:
I think we could do this natively within coverage by putting:

```
[run]
relative_files = true
```

in our .coveragerc config. See https://github.com/nedbat/coveragepy/issues/948

Reply:
This was a while back, but I'm pretty sure I tried this and it failed for me. But I should test this again before merging.